site/docs/components/progress/index.mdx

```diff
@@ -1,23 +1,21 @@
 ---
 title: Progress
+sidebar:
+  label: Progress 🚧
 data:
-  description: "A progress indicator gives the user an understanding of how long a system operation will take. It should be used when the operation will take more than a second to complete. Two variants are available to accommodate different layout constraints: linear and circular."
+  description: "CircularProgress and LinearProgress communicate the status of an ongoing operation, such as file downloads or data uploads. The two components accommodate different layout constraints."
```

Comment:
```suggestion
  description: "Circular Progress and Linear Progress communicate the status of an ongoing operation, such as file downloads or data uploads. The two components accommodate different layout constraints."
```
would this work better since they aren't in code